src/eventkit.cs

```diff
@@ -319,11 +328,13 @@ interface EKEvent {
 		NSDate OccurrenceDate { get; }
 
 #if MONOMAC
```

Comment:
Any reason to keep the `#if MONOMAC` here?

Reply:
I thought I did, but I don't see why now. Let me try removing it.

Reply:
Done!